mmv1/third_party/terraform/services/apphub/data_source_apphub_discovered_workloads.go

```diff
@@ -0,0 +1,177 @@
+package apphub
+
+import (
+    "fmt"
+    "time"
+
+    "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+    "github.com/hashicorp/terraform-provider-google/google/tpgresource"
+    transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func DataSourceApphubDiscoveredWorkload() *schema.Resource {
+    return &schema.Resource{
+        Read: dataSourceApphubDiscoveredWorkloadRead,
+        Schema: map[string]*schema.Schema{
```

Comment:
Should `project` be added into schema and be required?

Reply:
As `project` is required in the GET endpoint, I think it is required in the schema of this datasource. Which PR is it included? I wonder if `project` is included for other resources in the different PR instead of for this data source.

Reply:
Done.